src/components/Footer/sgds-footer.ts

```diff
@@ -13,36 +13,75 @@ export interface ColumnLinks {
 }
 
 /**
+ * @summary The footer contains supporting information for your service at the bottom of your website. All .gov.sg digital services shall contain a Global Footer Bar across all pages. The Global Footer Bar should include the name of the digital service, contact information, a privacy statement and the terms of use.
+ *
  * @csspart footer-top - The component's footer-top section container.
  * @csspart footer-bottom - The component's footer-bottom section container.
+ *
+ * @cssproperty footer-top - The component's footer-top section container.
+ * @cssproperty footer-bottom - The component's footer-bottom section container.
  */
 @customElement("sgds-footer")
 export class SgdsFooter extends SgdsElement {
   static styles = [SgdsElement.styles, styles];
 
-  @property()
-  title = ``;
-  @property()
-  description = ``;
+  /**
+   * 	Sets title of SgdsFooter
+   */
+  @property({ type: String })
+  title: string;
+
```

Comment:
i thought we discuss that anything visual can have a default value of empty string

Reply:
like this? 
```suggestion
 @property({ type: String })
  title = ""
```

Reply:
yess

Reply:
okay i see what u did with description. you want to remove the entire div.description when description is not available. 

can follow same approach for title to standardized

Reply:
`title` is mandatory so will keep it

Reply:
ohh ic okay then !